api/src/test/unit/java/uk/gov/dvsa/mot/trade/read/core/TradeReadServiceTest.java

```diff
@@ -563,6 +568,127 @@ public void getLatestMotTestByMotTestNumberWithSameRegistrationAndVin_VehicleWit
         assertEquals("Secondary colour is incorrect", secondaryColour, apiVehicle.getSecondaryColour());
     }
 
+    @Test
+    public void getLatestMotTestByDvlaVehicleId_ReturnsVehicle_FullyPopulated_WhenMotTestExists() {
+
+        final Integer dvlaVehicleId = 123;
```

Comment:
Happy to leave it the way it currently is :)  In this case re-use would probably make the tests more complicated